test/unit/user_test.rb

```diff
@@ -4,7 +4,9 @@ class UserTest < ActiveSupport::TestCase
   include AwsDecrypt
 
   setup do 
-   @user = crank(:user)
+   @school = crank!(:school)
+   crank!(:user, :school_id => @school.id)
```

Comment:
I think you can remove this line.

Reply:
We cannot. Reason being, if the test database is empty, the method called assign_barcode will throw an error since there is a query in that method to return a user record from the database.